bubblewrap.c

```diff
@@ -2201,8 +2207,11 @@ main (int    argc,
   if (proc_fd != -1)
     close (proc_fd);
 
-  if (opt_sync_fd != -1)
-    close (opt_sync_fd);
+  if (!no_reaper)
```

Comment:
I guess that leaking the fd should work. However, please add a more detailed comment about this in the code.